train_on_strand.sh

```diff
@@ -11,7 +11,7 @@
 #SBATCH --cpus-per-task=20
 
```

Comment:
Please untrack this file

Reply:
I did not manage to untrack the file, so  I just copied the train_on_strand from the main branch into mine. Now, there should not be any conflict anymore.